### PR TITLE
Add node 8, update README

### PR DESCRIPTION
**Pull request by netes**
This PR adds image for Node 8 🎉
It's basically almost the same as Node 6 but without `bower` which is not really necessary (and outdated).

### CODE REVIEW

File: node/README.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ Quay repository: https://quay.io/repository/netguru/ng-node
 
 ## Description
 
-This is a base image that can be used to build and run SPA apps. Contains couple most-necessary software, like:
+There are base images that can be used to build and run SPA apps. Contains couple most-necessary software, like:
```
Comment (changs) [inline node/README.md]: `These`?

Comment (netes) [inline node/README.md]: Typo :D

**Comment by netes** [comment]
I was thinking... maybe we should also change Yarn installation method to `apt-get` instead of `npm`? WDYT?

**Comment by rindek** [comment]
@netes  you're right, I'm actually installing YARN that way in baseimage: https://github.com/netguru/ng-dockerfiles/blob/master/baseimage/0.10.0/Dockerfile#L33-L37

I'll fix your branch with that code and merge it

**Comment by rindek** [comment]
The quay.io/netguru/ng-node:8 is currently being uploaded...

**Comment by rindek** [comment]
`quay.io/netguru/ng-node:8` is ready to be used

**Pull request merged**